01/das-dev/wordvalue.py

```diff
@@ -0,0 +1,14 @@
+from data import DICTIONARY, LETTER_SCORES
+
+def load_words():
+    with open(DICTIONARY, 'r') as f:
+        return f.read().split()
```

Comment:
does this strip the newline of each line?

Reply:
> does this strip the newline of each line?

Is my code not correctly?

Reply:
It does work: 'print(f.read().split()[:10])' returns `['A', 'a', 'aa', 'aal', 'aalii', 'aam', 'Aani', 'aardvark', 'aardwolf', 'Aaron']`

Reply:
I am just used to writing `[line.strip() for line in f.readlines()]`